### PR TITLE
feat: 2px slideshow section

### PR DESCRIPTION
**Pull request by LHDi**

### CODE REVIEW

**Review by freadbaker — CHANGES_REQUESTED** [review]
@LHDi This looks good! Two things I missed from the Figma prototype, though: 

1. The ability to go to next slide when clicking anywhere in the screen
2. The ability to control the slides with the arrow keys on my keyboard.

**Comment by LHDi** [comment]
> The ability to go to next slide when clicking anywhere in the screen
is this addressed anywhere @freadbaker 🤔

**Comment by LHDi** [comment]
@freadbaker please take another look 🙏

File: apps/web/vibes/2px/components/slideshow-section/index.tsx [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+                height={800}
+                className="h-auto max-h-full w-auto max-w-full cursor-pointer border-2 border-foreground object-contain"
+                onClick={() => setCurrentSlideIndex(i)}
+              />
+            </div>
+          ) : null
+        )}
+      </div>
+      <div className="mx-auto mb-10 w-full max-w-xl px-5 text-center text-sm text-foreground">
+        <h2 className="mb-2 font-mono leading-[1.375rem] tracking-[0.02em]">
```
Comment (freadbaker) [inline apps/web/vibes/2px/components/slideshow-section/index.tsx:116]: This should be uppercase.

**Review by freadbaker — APPROVED** [review]
Great work!

**Pull request merged**